### PR TITLE
Updating runtime prereqs

### PR DESCRIPTION
**Pull request by Petermarcu**
# Title

On the title describe
what you've fixed (or created) with this Pull Request (PR).

## Summary

Insert a short (one or two sentence) summary here.

Fixes #Issue_Number

>Note: The "Fixes #nnn" syntax in the PR description causes
>GitHub to automatically close the issue when this PR is merged.
> Remove that line if you don't have issues associated with this
> PR. Click on the Guidelines for Contributing link above for details.

## Details

Explain your changes, and why you made them. If that
information is already available in the issue referenced
above, just referencing the issue is preferred to copying
the text.

This may not be necessary depending on the scope of the PR 
changes. For example, "fix typo in introduction.md" is
sufficient to describe that PR.

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

**Comment by Petermarcu** [comment]
@janvorli

File: docs/core/linux-prerequisites.md [hunk 1]
```diff
@@ -58,33 +58,35 @@ See [.NET Core 1.x Supported OS Versions](https://github.com/dotnet/core/blob/ma
...
-* unzip
+* liblttng-ust0
+* libcurl3
+* libssl1.0.0
+* libuuid1
+* libkrb5
+* zlib1g
+* libicu52 (for 14.X)
+* libicu55 (for 16.X)
+* libicu57 (for (17.X)
```
Comment (janvorli) [inline docs/core/linux-prerequisites.md]: A nit - typo - extra parenthese

**Review by janvorli — APPROVED** [review]

**Review by rpetrusha — APPROVED** [review]
Thanks for the update, @Petermarcu. We'll merge as soon as a build completes successfully.

**Comment by rpetrusha** [comment]
Build failed for an unspecified reason. Closing and reopening to begin new build.

**Comment by mairaw** [comment]
Build looks good. Thanks @Petermarcu. The fix should go live later today.

**Pull request merged**